spec/classes/collectd_plugin_logfile_spec.rb

```diff
@@ -24,8 +24,7 @@
   context ':ensure => present, specific params, collectd version 4.9' do
     let :facts do
       { osfamily: 'Redhat',
-        collectd_version: '4.9.0'
-      }
+        collectd_version: '4.9.0' }
```

Comment:
this is really wonky. can you migrate that to:
```ruby
{
  osfamily: 'Redhat',
  collectd_version: '4.9.0'
}
```

also applies for the other hashes.